lib/collectors.js

```diff
@@ -44,29 +44,35 @@ module.exports = (api) => {
     }
   };
 
-  api.metasync.DataCollector.prototype.on = function(
-    // DataCollector events:
-    eventName,
-    callback
-    // on('error', function(err, key))
-    // on('timeout', function(err, data))
-    // on('done', function(errs, data))
-    //   errs - hash of errors
-    //   data - hash of sucessfully received adta
-  ) {
-    if (eventName in this.events) {
-      this.events[eventName] = callback;
-    }
-  };
+  const Collector = {
+    on(
+      // Collector events:
+      eventName,
+      callback
+      // on('error', function(err, key))
+      // on('timeout', function(err, data))
+      // on('done', function(errs, data))
+      //   errs - hash of errors
+      //   data - hash of sucessfully received adta
+    ) {
+      if (eventName in this.events) {
+        this.events[eventName] = callback;
+      }
+    },
```

Comment:
I think comma after last element simplifies swapping, adding and deleting elements because you shouldn't think about which element is followed by comma and which is not (each element is followed). 
But if code style doesn't apply this, I will code without last element comma.